### ISSUE TITLE
Network Adapter Type

### ISSUE THREAD

**Issue by infabo** [AOEpeople/chef-devbox#1]
Hi Fabrizio!

I faced super terrible Samba transfer speeds using your unibox here (Windows 8.1). About 600kb/s to max. 2mb/s. PhpStorm Indexing files at startup was painfully slow too.
Virtual Box network adapter emulation is - what I have read - very poor with host-only adapter using one of the types like "Intel PRO/1000 MT Desktop" (which is default) on Windows.

Please add this to your Vagrantfile:

```
    v.customize ["modifyvm", :id, "--nictype1", "virtio"]
    v.customize ["modifyvm", :id, "--nictype2", "virtio"]
```

My transfer speed increased dramatically. I'm now at ~80m/s.

**Comment by fbrnc** [comment]
Hi @infabo,

I've added those lines two weeks ago and observed the performance since then. While I didn't do any real speed tests I can't tell this is any faster (and especially not significantly faster). But also I didn't feel it was very slow in the first place.
So, I think these lines didn't have any effect in my setup.
Since I don't really understand what they're supposed to do and why this might speed up the network adapter I'll only add it commented out.

What's your experience after two weeks? Can you still confirm that those lines made a difference? Do you know what's exactly happening and why this might not have any effect on other systems? Is it because other systems already come with a better network adapter support for some other reason?

**Comment by infabo** [comment]
> I'll only add it commented out.

Yes, that's a good idea. It may not be the best idea to set these options per default. It is just nice to have it in there commented out (maybe with some hint like "uncomment if you experience slow transfer speed").

I didn't do any speed tests either since then, but I'm still not happy.
With the first databag I set up on the devbox (without any vagrant settings changed) I was very pleased with. The samba share and the performance in general was great. I copied several large tar archives with ease and general access speed (Browser, Magento backend, etc.) was like I would expect it from my other local VMs.
But suddenly the speed issues appeared out of the blue. Copy dropped to <2mb/s and this was not acceptable. I experimented with the settings and nictype "virtio" did help.
While the sequential transfer speed increased dramatically, accessing files on the samba share (e.g. with an editor) is still painfully slow. Simple text files need 4 to 6 seconds to open.

**Comment by infabo** [comment]
Hi @fbrnc 

today I have upgraded to VirtubalBox 4.3.26 and reverted my changes to the network adapter type (virtio -> Intel Pro/1000 MT Desktop) - et voila! I get 140mb/s over samba and random access is super-fast again.
Seems like I had troubles with my VirtualBox installation or they were related to the VirtualBox version I used before.
